package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "p2p-money",
+  "version": "1.0.0",
+  "description": "An app to help people exchange money online. without a middleman like a bank or exchange agency",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\""
```

Comment:
delete this line, 
we don't have test yet

Reply:
I will do it in issue #30 
we add file structure without edit script
@ahmadysalah